src/reducers/userDataReducer.js

```diff
@@ -55,6 +56,17 @@ function userDataReducer(state = initialState, action) {
       //   })
       return state;
     case SET_MOST_RECENT_PROGRAM:
+      console.log(action.value);
+      fetch
+        .updateUserData(state.uid, { mostRecentProgram: action.value })
+        .then(response => {
+          console.log(action.value);
+          console.log(response);
+        })
+        .catch(err => {
+          state.error = err;
+          console.log(err);
```

Comment:
I think this one is ok, bc it's always good to know when stuff goes wrong if we're tryna debug in the field